virus/virTemplate.md

```diff
@@ -1,40 +1,106 @@
+
+In addition to dataset ERR3143221, the Posasimilis Intestiviridae was also found in dataset ERR3143231, which belongs to the same study by Kauer et al. Notably, the source of this hit was Ovis aries (sheep) fecal matter rather than goat. Ensuing Blastx alignment tests match the candidate RdRp sequence with Caudovirales sp., Lachnospiraceae bacterium, and Bacteroidales bacterium as the top hits. These hits are peculiar as the percent identity score is two-fold greater compared to rasavirus sp. of the previous analysis. Furthermore, these hits are vastly different than posa-like viruses. For instance, Caudovirales sp. is species of virus within the Caudovirales order, a classification of double-stranded DNA bacteriophages [Tisza, 2021]. It could be that Posasimilis intestinum is an RNA bacteriophage. However, the known classes of RNA bacteriophages to date, Leviviridae (single-stranded RNA viruses) [Tars, 2020] and Cystoviridae [Mantynen, 2023], were not among the alignment match hits. Nonetheless, it remains a possibility that our virus of interest is in actuality an RNA bacteriophage that is evolutionarily related to double-stranded DNA bacteriophages. This proposal is supported by the Lachnospiraceae and Bacteroidales bacterium matches. Lachnospiraceae is abundant in the digestive tracts of mammals, humans included [Meehan, 2014], and Bacteroidales have been documented in the gastrointestinal tracts of homeothermic animals [Ormerod, 2016]. These bacteria are considered members of their host microbiome and could act as an avenue for which the virus of interest penetrates into the resulting fecal matter. The theory that posaviruses may infect a third-party rather than the host itself has previously been explored. Blast analysis of posavirus sequences has shown similarity to RNA sequences from a sample of the parasite Ascaris suum [Munnink, 2017]. 
+
+The proposal for a bacterial-mediated entry into the host organism is reflected by the results from another dataset, SRR5830110 (accessible via the SRA database). The sample containing the Posasimilis Intestiviridae hit derives from the fecal community of Tule Elk. Blastx analysis matches the virus with Caudovirales sp. and Candidatus Scalindua sp. as the top hits. The presence of Caudovirales may strengthen the notion that our virus of interest operates in a bacteriophage-like manner. Moreover, Candidatus Scalindua is a genus of marine-associated bacteria first discovered in a waster water treatment plant [Schmid, 2003]. This highlights the capacity for Posasimilis intestinum to infect bacteria, specifically bacteria that may be found in the water supply of wild-life. 
+
+All figures and related information pertaining to dataset runs and Posa can be found in the "MGY441 A3 Virus Seq Runs" csv file.
+[Uploading MGY441 A3 Virus Seq Runs.csv…]()
+[MGY441 A3 Virus Seq Runs.csv](https://github.com/DerekTea/VIRUSxDISCVRY/files/13552458/MGY441.A3.Virus.Seq.Runs.csv)
+
 ### Ecology subsection A...
```

Comment:
Give this a more informative title fitting to a report, not a "homework assignment"